src/components/app.jsx

```diff
@@ -174,6 +218,16 @@ const App = React.createClass({
 
           <footer className="w-100 pa3 tc white">
             <FooterButton label="Generate code" onClick={this.openCodeModal} />
+            <div style={ inlineBlock }>
+              <FooterButton label={colorPickerLabel} onClick={this.openColorPicker} />
+              { this.state.colorPickerShowing
+                ? <div style={ cover }><SketchPicker
```

Comment:
if you wrap this with `<div style={{ position: 'relative' }}>...</div>`, it will follow the button when everything is in one column, i.e.:

![](https://i.imgur.com/fNVQgcL.png)

the `bottom` property in `cover` needed to be adjusted a bit when i did this, though. `45px` looked okay.